### ISSUE TITLE
Endpoint offset issue

### ISSUE THREAD

**Issue by aegissniper** [jsplumb/jsPlumb#668]
![image](https://user-images.githubusercontent.com/26962190/30036182-a0ce3c66-91b9-11e7-8354-1e45c4009601.png)
The connection endpoint is offset when it is showing on bottom of the item.
And sometimes the connection starts from blank space instead of start endpoint while dragging from endpoint.

Please help me

**Comment by sporritt** [comment]
have you got any `margin` set on those elements?

**Comment by aegissniper** [comment]
No, I didn't set any margin. I checked it on chrome and the margin is 0.
It happens only in case of the bottom endpoint.
And sometimes when I create a connection between two nodes, the start endpoint is far away from the actual endpoint.

I think these issues may be same.

**Comment by sporritt** [comment]
you've got the Container set correctly?

**Comment by aegissniper** [comment]
yeah, I think so. Is there any specific way to check it.
![image](https://user-images.githubusercontent.com/26962190/30134975-d05f41f6-9361-11e7-80de-d14b1190f4a8.png)

The offset appears for the bottom endpoint also

**Comment by aegissniper** [comment]
@sporritt It is working occasionally. I avoided using bottom endpoint. And have another issue.
![image](https://user-images.githubusercontent.com/26962190/30288399-3f92a39c-9731-11e7-807d-26347a51e3ea.png)
At initial loading, I added nodes and made connections between nodes.
Unfortunately, the connection edges are offset from the target node and they became connected after moving nodes a bit.

**Comment by KartheekJavvaji** [comment]
try updating the offset of the endpoints and repainting the element after the drag-stop event
